draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1991,21 +1991,21 @@ CachedInformationType Values" registry that was defined in [RFC7924].
 IANA is asked to establish a registry of hash algorithm values, named
 "CT Hash Algorithms", that initially consists of:
 
-|-------+---------------------|
-| Index | Hash                |
-|-------+---------------------|
-| 0     | SHA-256 [RFC4634]   |
-| 255   | reserved            |
-|-------+---------------------|
-
-An expert review for adding values to this registry, to comply with the
-"Expert Review" policy described in [RFC5226].
+|-------------+----------------+------------------------------------------|
+| Value       | Hash Algorithm | Reference / [RFC5226] Assignment Policy  |
```

Comment:
Nit: Move the [RFC5226] reference from the column header and into the beginning of this section, saying something along the lines of:
"The Specification Required and Expert Review criteria mentioned in this section refer to the policies specified in [RFC5226]".